src/common/services/casApi.service.js

```diff
@@ -0,0 +1,69 @@
+import angular from 'angular';
+import 'angular-environment';
+import {Observable} from 'rxjs/Observable';
+import 'rxjs/add/observable/from';
+import 'rxjs/add/operator/map';
+import 'rxjs/add/operator/do';
+
+let serviceName = 'casApiService';
+
+class CasApi {
```

Comment:
I don't know all the cas endpoints yet, I do know there will be sign-in, sign-out, forgot password and possibly the security questions. But nothing other than sessionService will use it, so it makes sense to just do $http in the service.